File: class03/cpp/AP0301-SaqueBancario.cpp
#include <iostream>
#include <iomanip>
int main() {
    double saldo, saque;
    std::cout << "Saldo: R$ "; std::cin >> saldo;
    std::cout << "Saque: R$ "; std::cin >> saque;
    if (saque > saldo) std::cout << "Saldo insuficiente" << std::endl;
    else if ((int)saque % 10 != 0) std::cout << "Use multiplos de R$10" << std::endl;
    else { saldo -= saque; std::cout << "Saque liberado! Saldo: R$ " << std::fixed << std::setprecision(2) << saldo << std::endl; }
    return 0;
}

File: class03/python/AP0301-SaqueBancario.py
saldo = float(input("Saldo: R$ "))
saque = float(input("Saque: R$ "))
if saque > saldo:
    print("Saldo insuficiente")
elif saque % 10 != 0:
    print("Use múltiplos de R$10")
else:
    saldo -= saque
    print(f"Saque liberado! Saldo: R$ {saldo:.2f}")
